typescript/web/src/pages/posts/[slug].tsx

```diff
@@ -42,6 +47,12 @@ export default function Posts({
               ]
             : undefined
         }
+        canonical={`${origin}/posts/${article?.slug}`}
+        article={{
+          authors: [article?.author?.name],
```

Comment:
If `article?.author?.name` is `undefined`, you'll have `authors: [undefined]` instead of `authors: []`.
I suggest replacing the value above by either `article?.author?.name ? [article.author.name] : []`

Reply:
Done in 016245f93d83a6f4052af475da30017be328c70c